include/battle_main.h

```diff
@@ -57,7 +57,7 @@ extern const u8 gStatusConditionString_IceJpn[8];
 extern const u8 gStatusConditionString_ConfusionJpn[8];
 extern const u8 gStatusConditionString_LoveJpn[8];
 extern const u8 *const gStatusConditionStringsTable[7][2];
-extern const u8 gTypeEffectiveness[405];
+extern const u16 gTypeEffectiveness[441];
```

Comment:
Why change this to `u16`?

Reply:
Oh it was being fucky and I tried that first to resolve it - Turns out it was cuz I didn't maintain the [441] elsewhere, but making it a u16 didn't seem to break anything. I'll change it back cuz I forgot to LOL